backend/src/Altinn.Authorizationadmin/Altinn.Authorizationadmin/Services/Implementation/PolicyAdministrationPoint.cs

```diff
@@ -405,14 +423,11 @@ private async Task<List<Rule>> ProcessPolicyFile(string policyPath, string org,
 
         private async Task<List<Rule>> DeleteAllRulesInPolicy(RequestToDelete policyToDelete)
         {
+            string altinnAppId = null;
+            ServiceResource resource = null;
             DelegationHelper.TryGetResourceFromAttributeMatch(policyToDelete.PolicyMatch.Resource, out ResourceAttributeMatchType resourceMatchType, out string resourceRegistryId, out string org, out string app);
             string coveredBy = DelegationHelper.GetCoveredByFromMatch(policyToDelete.PolicyMatch.CoveredBy, out int? coveredByUserId, out int? coveredByPartyId);
 
-            if (resourceMatchType != ResourceAttributeMatchType.AltinnApp)
```

Comment:
Still should check the result of the TryGetResourceFromAttributeMatch operation. If it fails and/or the resourceMatchType is none operation should fail as the resource cannot be identified